### ISSUE TITLE
(urn:cts:greekLit:tlg1264.tlg002.1st1K-grc1) new edition

### ISSUE THREAD

**Issue by ahanhardt** [OpenGreekAndLatin/First1KGreek#2471]
Please see Trello cards for zip file and names of volunteers:

[Publishing Tasks card](https://trello.com/c/TQmNAuD9/115-publish-stoicorumveterum03arniuoft)
[Editing Tasks card](https://trello.com/c/gnCtJwDm/114-edit-stoicorumveterum03arniuoft)

**Comment by charhoughton** [comment]
claiming for initial markup

**Comment by charhoughton** [comment]
@lcerrato what's the best way to format what appears to be a verse fragment? See here: https://archive.org/details/stoicorumveterum03arniuoft/page/96/mode/1up, frag. 397. It doesn't look like a quotation to me. It's just not something I've come across before, so I wanted to check.

**Comment by charhoughton** [comment]
@lcerrato initial markup and formatting completed. It was a big file and needed a lot of restructuring, so if there are big things I missed or need to fix, please let me know and I'll be happy to take another close look at it.
[tlg1264.tlg002.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/7198362/tlg1264.tlg002.zip)

**Comment by lcerrato** [comment]
@charhoughton 
Thank you! I'll take a look. I am working on similar files now, with a lot of interspersed different languages and bracketed info so I know what a pain this is.

At first glance, we probably want `head` instead of `label` and some of the labels are greek but mis-tagged latin. I tend to restrict labels to non-text info that is digits or section numbering in Greek or something similar. If things are centered on the page, they are generally headers. I know some of these span many sections but I think we should just not worry about that.

I will address these globally when I finish the conversion.

(Also be sure to remove all comments `<!-- --> ` in the headers.)

The most noticeable thing I see is when another bibliographic reference occurs in the same div, so in div 11, I can see 

https://archive.org/details/stoicorumveterum03arniuoft/page/4/mode/2up

`... － － 25. Sapientia enim et
          <pb facs="stoicorumveterum03arniuoft_0013"/>
          animi magnitudinem complectitur et iustitiam et ut omnia,
          quae homini accidant, infra se esse iudicet.</p>
          </div>`

Here, the － － 25. is indicating that this is a new reference, following the `<bibl>Cicero de finibus III 23.</bibl> ` which is at the start of fragment 11, so this means, in effect, **Cicero de finibus III 25**

So these digits are references and should be tagged as `<bibl>`. Normally, I would be flagging and removing stray digits, but these seem to be important to this text? In fragment 13 I also see `....15. Hoc sic expositum...`

I'll do these steps with the check in. Just be on the lookout in the future.

**Comment by charhoughton** [comment]
@lcerrato many thanks! I think some of the other bibliographic references within divs were tagged as ```<bibl>``` but I know I didn't get ones that were just numbers. As for when to use ```<label>``` vs <```head>```, thanks for letting me know about that. The headers in this text are definitely a pain with the fragment divs!

**Comment by lcerrato** [comment]
@charhoughton 
Yes, it's hard to know what a random digit (or a dash and digit, or bracketed digit) is doing in a file until you dive in. Fortunately, I can locate these pretty easily and try to tag them. 
Plus, it's **not** the end of the world if they aren't fixed in any given pass.....

**Comment by lcerrato** [comment]
+72,251 words!
